packages/dds/tree/src/test/simple-tree/arrayNode.spec.ts

```diff
@@ -590,4 +590,28 @@ describe("ArrayNode", () => {
 			);
 		});
 	});
+
+	describe("Iteration", () => {
+		it("Concurrently iterating and editing should throw an error.", () => {
+			assert.throws(
+				// False positive
```

Comment:
I'm confused by this comment. To me this implies that the error detection here is a false positive, meaning its detecting an error when it should not. This case however should (and if this test passes) does error, so true positive for the detection.

Reply:
removed incorrect comment